### PR TITLE
Expand mapping tutorial and add workflow + tests

### PR DESCRIPTION
**Pull request by bebatut**
- Add more details about mapping theory
- Add more steps for BAM exploration
- Update the steps

### CODE REVIEW

File: topics/sequence-analysis/tutorials/mapping/igv.md [hunk 1]
```diff
@@ -9,14 +10,21 @@
...
 > {: .solution }
 {: .question}
 
-> ### {% icon comment %} Comments
-> Because the number of reads over a region can be quite large, the IGV browser by default only allows to see the reads that fall into a small window. This behaviour can be changed in the IGV from `view > Preferences > Alignments`.
-{: .comment}
+> ### {% icon tip %} Tips for IGV
+> 1. Because the number of reads over a region can be quite large, the IGV browser by default only allows to see the reads that fall into a small window. This behaviour can be changed in the IGV from `view > Preferences > Alignments`.
+> 2. If the genome of your interest is not there check if it is available via **More...**. Is this is not the case you can add it manually via the menu **Genomes** -> **Load Genome from...**
```
Comment (nsoranzo) [inline topics/sequence-analysis/tutorials/mapping/igv.md]: s/Is this is not the case you can/If this is not the case, you can/

File: topics/sequence-analysis/tutorials/mapping/igv.md [hunk 2]
```diff
@@ -9,14 +10,21 @@
...
+>
+>    ![Select genome in IGV](../../images/igv_select_genome.png "Select genome in IGV")
+>
+> A general description of the user interface of the IGV browser can be read here: [IGV Browser description]({{site.baseurl}}/topics/introduction/tutorials/igv-introduction/tutorial.html)
```
Comment (nsoranzo) [inline topics/sequence-analysis/tutorials/mapping/igv.md]: s/can be read/can be found/ (or "is available")

File: topics/sequence-analysis/tutorials/mapping/ref_genome_explanation.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+We need a reference genome on which we map the reads.
+
+> ### {% icon question %} Questions
+>
+> 1. What is a reference genome?
+> 2. For human, several possible reference genomes are available: hg18, hg19, hg38, etc. What do they correspond to?
+> 2. Which reference genome should we use?
+>
+> > ### {% icon solution %} Solution
+> > 1. A reference genome (or reference assembly) is a DNA sequence assembled as a representative example of a species' DNA sequence. As they are often assembled from the sequencing of the DNA from different donors, they do not accurately represent the set of genes of any single person but a mosaic of different DNA sequences from each donor.
```
Comment (nsoranzo) [inline topics/sequence-analysis/tutorials/mapping/ref_genome_explanation.md]: Trying to be as general as possible (e.g. considering RNA viruses):

s/is a DNA sequence/is a set of nucleic acid sequences/
s/a species' DNA sequence./a species' genetic material./
s/from the sequencing of the DNA from different donors/from the sequencing of different individuals/
s/person/organism/
s/different DNA sequences from each donor./different nucleic acid sequences from each individual./

**Comment by bebatut** [comment]
Thanks @nsoranzo for the comments

**Pull request merged**